hrm-service/permission-rules/cl.json

```diff
@@ -0,0 +1,22 @@
+{
+  "closeCase":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"reopenCase":  [["isSupervisor"]],
+"caseStatusTransition":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addNote":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editNote":  [["isSupervisor"]],
+"addReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editReferral":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editHousehold":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editPerpetrator":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editIncident":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"addDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editDocument":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editCaseSummary":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editChildIsAtRisk":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editFollowUpDate":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
+"editContact":  [["isSupervisor"],["isCreator", "isCaseOpen"]],
```

Comment:
For editing contacts, i.e., "editContact" does not recognize "isCreator" and "isCaseOpen" as conditions. Instead of ["isCreator", "isCaseOpen"], ["isOwner"] will work.